vectordatasource/transform.py

```diff
@@ -137,7 +138,11 @@ def _to_int_degrees(x):
         'west':  270, 'W': 270, 'WNW': 292, 'NW': 315, 'NNW': 337
     }
 
-    return cardinals[x]
+    # protect against bad cardinal notations
+    if x in cardinals:
```

Comment:
This:

```python
if x in cardinals:
  return cardinals[x]

return None
```

Is equivalent to:

```python
return cardinals.get(x)
```

Reply:
Fixed in https://github.com/tilezen/vector-datasource/pull/1921/commits/54b33711937fddde00a84322c739c9fbfe536b43.